### ISSUE TITLE
Issue in capturing two date fields in a sentence

### ISSUE THREAD

**Issue by Srivishnu27feb** [facebook/duckling#691]
Hi Team,

I face issue in extracting two date field by using duckling extractor:

curl -XPOST http://.0.0.0.0:8000/parse --date 'locale=en_GB&text=Block dates from today to tomorrow'

The date for today is getting captured correctly as '2022-05-12' but tomorrow is captured as '2022-05-14' (t+1), and the same issue is happening when i try it out with actual dates instead of today or tomorrow (The second extracted entity is always dt+1). Please can you help me with the fix.

**Comment by chessai** [comment]
this is correct because time intervals in duckling are always lower inclusive, upper exclusive.
